docs/source/usage.rst

```diff
@@ -11,20 +11,21 @@ You can find a full list of variables in `env.example`:
 
 .. literalinclude:: ../../env.example
 
-You can set then load all of these variables at once in the command line:
+If you have created your own .env file, mirar will try to automatically load the .env file.
+
+You can also set individual variables in the command line:
 
 .. code-block:: bash
 
-    set -o allexport
-    source .env
-    set +o allexport
+    export RAW_DATA_DIR=/home/astronomer/rawdata
 
-Or you can also set individual variables in the command line:
+Or load all of these variables at once using the command line:
```

Comment:
I think this bit is now redundant, right? If you have the dotenv file, it'll load. Or is this when the dotenv file is somewhere else than code e.g if installed from pypi?

Reply:
Yep, it's for when the dotenv file is somewhere else.

Reply:
Can you mention that explicitly in the text?